core/src/test/java/apoc/refactor/rename/RenameTest.java

```diff
@@ -55,6 +55,55 @@ public void testRenameLabel() throws Exception {
 		assertEquals(0L, resultNodesMatches("Foo", null));
 	}
 
+	@Test
+	public void testRenameLabelDoesntAllowCypherInjection() throws Exception {
+		db.executeTransactionally("UNWIND range(0,9) AS id CREATE (f:Foo {id: id})");
+		testCall(db, "CALL apoc.refactor.rename.label(" +
+						"  'Foo', " +
+						"  'Whatever` WITH n MATCH (m) DETACH DELETE m //'" +
+						")",
+				map(),
+				(r) -> {});
+
+		assertEquals(10L, resultNodesMatches("`Whatever`` WITH n MATCH (m) DETACH DELETE m //`", null));
+		assertEquals(0L, resultNodesMatches("Foo", null));
```

Comment:
It seems that there we are wrapping the entire label in more "`" than needed.

```
CALL apoc.refactor.rename.label("Foo", "Whatever")  // renames label "Foo" to "Whatever"
CALL apoc.refactor.rename.label("Foo", "What`ever") // rename label "Foo" to "What`ever", not "`What`ever`"
```

Thus in your test I would expect 
```
resultNodesMatches("`Whatever`` WITH n MATCH (m) DETACH DELETE m //`", null)); //  == 0
```

Apologies if I'm missing something, please feel free to reach out on Slack

Reply:
Oh, no it is just the test function resultNodesMatches, didn't add wrapper backticks so this would have tried to find nodes called `node.Whatever`` WITH  n MATCH (m) DETACH DELETE m //` which isn't valid cypher (the spaces break it), would you prefer me to change the tester function to add backticks everywhere for consistency?

Reply:
Oh sorry that was my confusion then! No need to change it